yarn.lock

```diff
@@ -4634,13 +4634,15 @@ handlebars@^4.0.1:
   optionalDependencies:
     uglify-js "^3.1.4"
 
-handlebars@^4.5.1:
-  version "4.5.1"
-  resolved "https://registry.yarnpkg.com/handlebars/-/handlebars-4.5.1.tgz#8a01c382c180272260d07f2d1aa3ae745715c7ba"
+handlebars@^4.5.3:
+  version "4.7.6"
```

Comment:
Looks like we are actually going to version 4.7.6

Reply:
Do you want to lock to specific version? The command I ran is `yarn upgrade handlerbars@^4.5.3`, which allows latest minor update to come in.